run.sh

```diff
@@ -3,7 +3,7 @@
 /usr/sbin/sshd -D &
 
 $HADOOP_PREFIX/sbin/hadoop-daemon.sh start namenode
-
+$HADOOP_PREFIX/sbin/start-dfs.sh
```

Comment:
@alex-ber , this is running with `dfs.sh` why are you re-running it here?

Reply:
It is running in different phases. dfs.sh run when we build the image, run.sh is run when we spin the container. I spent quite a lot of time their, as far I remember if I will remove it will stop to work (the commans below will fail, maybe silently). Do you want me to double check this?

Reply:
Please double check

Reply:
Ok.

Reply:
It doesn't work.

org.apache.hadoop.ipc.RemoteException(java.io.IOException): File /user/root/input/yarn-site.xml._COPYING_ could only be replicated to 0 nodes instead of minReplication (=1).  There are 0 datanode(s) running and no node(s) are excluded in this operation.

I'll leave this line inplace.